modules/ob-about-build-strategies.adoc

```diff
@@ -0,0 +1,40 @@
+// Ths module is included in the following assembly:
+//
+// * builds/overview-openshift-builds.adoc
+
+:_content-type: CONCEPT
+[id="about-build-strategies_{context}"]
+= About build and cluster build strategies
+
+The `BuildStrategy` and `ClusterBuildStrategy` resources define a series of steps to assemble an application. You can use the `BuildStrategy` resources within a namespace and the `ClusterBuildStrategy` resources within a cluster.
+
+The specification of a `BuildStrategy` or `ClusterBuildStrategy` resource consists of a `steps` object. The following example shows the specification of the `Kaniko` cluster build strategy:
+[source,yaml]
+----
+apiVersion: shipwright.io/v1beta1
+kind: ClusterBuildStrategy
+metadata:
+  name: kaniko
+spec:
+  steps:
+    - name: build-and-push
+      image: gcr.io/kaniko-project/executor:v1.3.0
+      workingDir: /workspace/source
+      securityContext:
+        runAsUser: 0
+        capabilities:
+          add:
+            - CHOWN
+            - DAC_OVERRIDE
+            - FOWNER
+            - SETGID
+            - SETUID
+            - SETFCAP
+      env:
+        - name: PODMAN_CONFIG
+          value: /tekton/home/.podman
+      command:
+        - /kaniko/executor
+...
```

Comment:
```suggestion
# ...
```

Per [repo guidelines for YAML](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#formatting-1)